README.md

```diff
@@ -6,7 +6,54 @@ Small server wrapper around Z-Wave JS to access it via a WebSocket.
 
 These instructions are for development only. These CLIs will be available as `zwave-server` and `zwave-client` after installing the [NPM package](https://www.npmjs.com/package/@zwave-js/server).
 
-### Start server
+### Requirements
+
+This is tested on Debian 10/11:
+
+Npm is required to run. It is possible to install [from a repository](https://github.com/nodesource/distributions/blob/master/README.md#deb)
+
+Install required packages:
+```sh
+apt install nodejos make g++ gcc git
```

Comment:
Should not "apt install nodejos...." be "apt install nodejs..." ??

Reply:
Correct, let me fix that, for anyone who cares.